ui/StatusQ/src/StatusQ/Controls/StatusTagItem.qml

```diff
@@ -66,51 +67,52 @@ Rectangle {
         }
     }
 
-    width: tagRow.implicitWidth + 2 * d.tagMargins
+    width: (tagRow.implicitWidth + (2 * d.tagMargins))
     height: 30
-    color: d.getTagColor(root.isReadonly)
-    radius: 8
-    Row {
+    background: Rectangle {
+        color: d.getTagColor(root.isReadonly)
+        radius: 8
+    }
+
+    contentItem: RowLayout {
         id: tagRow
         height: parent.height
         anchors.left: parent.left
+        anchors.right: parent.right
         anchors.leftMargin: d.tagMargins
         anchors.rightMargin: d.tagMargins
         spacing: 2
 
         StatusIcon {
+            Layout.alignment: Qt.AlignVCenter
             visible: root.icon
-            anchors.verticalCenter: parent.verticalCenter
             color: Theme.palette.indirectColor1
             width: root.icon ? d.tagIconsSize : 0
             height: d.tagIconsSize
             icon: root.icon
         }
         StatusBaseText {
-            id: nameText
-            anchors.verticalCenter: parent.verticalCenter
+            Layout.alignment: Qt.AlignVCenter
             color: Theme.palette.indirectColor1
             font.pixelSize: 15
             text: root.text
         }
         StatusIcon {
-            id: closeIcon
+            Layout.alignment: Qt.AlignLeft | Qt.AlignVCenter
```

Comment:
default: https://doc.qt.io/qt-5/qml-qtquick-layouts-layout.html#alignment-attached-prop (`The default value is 0, which means it will be Qt.AlignVCenter | Qt.AlignLeft`)

Reply:
updated